tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py

```diff
@@ -128,11 +123,9 @@ def fluxes_from_temperature_full_domain(F, V, k):
 coupling_boundary = TopBoundary()
 bottom_boundary = BottomBoundary()
 
-coupling = Coupling(config_file_name, solver_name)
-coupling.set_coupling_mesh(mesh, coupling_boundary, coupling_mesh_name)
-coupling.set_read_field(read_data_name, u_D_function)
-coupling.set_write_field(write_data_name, f_N_function)
-coupling.initialize_data()
+precice = Adapter()
```

Comment:
This is weird. You call it `Adapter` and then `precice`. I would at least change `precice` here to `preciceAdapter`. Do you agree?

Reply:
same https://github.com/precice/fenics-adapter/pull/6#discussion_r226974367